pycm/pycm_param.py

```diff
@@ -59,7 +59,9 @@
 VECTOR_ONLY_ERROR = "This option only works in vector mode"
 VECTOR_UNIQUE_CLASS_ERROR = "The classes list isn't unique. It contains duplicated labels."
 CLASS_NUMBER_ERROR = "Number of the classes is lower than 2"
+CLASSES_ERROR = "Used classes is not a subset of matrix's classes."
 COMPARE_FORMAT_ERROR = "The input type is supposed to be dictionary but it's not!"
+CLASSES_LENGHT_ERROR = "Used classes has not the length equals to array length."
```

Comment:
I think we should change this error message!
@alirezazolanvari

Reply:
My suggestion is: "Classes length is not equal to the array length."